torch/onnx/symbolic_opset9.py

```diff
@@ -1944,8 +1944,21 @@ def to(g, self, *args):
             # aten::to(Tensor, Device, bool, bool, memory_format)
             return self
         else:
-            dtype = sym_help._maybe_get_const(args[0], 'i')
-            if sym_help._is_value(dtype):
+            # TODO: The logic here is weird for test/onnx/test_pytorch_onnx_onnxruntime.py::TestONNXRuntime::test_ones_bool
```

Comment:
```
torch.to(other, non_blocking=False, copy=False) → Tensor

    Returns a Tensor with same torch.dtype and torch.device as the Tensor other.
    When non_blocking, tries to convert asynchronously with respect to the host if possible, 
    e.g., converting a CPU Tensor with pinned memory to a CUDA Tensor.
    When copy is set, a new Tensor is created even when the Tensor already matches the desired conversion.
```

This signature takes another tensor, and cast to the same dtype (and device).

Reply:
The original code does not consider the case when `args[0]` is constant tensor (not int). This PR fixes it.